app/src/main/java/com/example/moviesapp/fragment/NowPlayingFragment.kt

```diff
@@ -20,6 +34,31 @@ class NowPlayingFragment : Fragment() {
         return binding.root
     }
 
+    private fun updateUI(movieData: NowPlayingViewModel.MovieData) {
+        when (movieData.state) {
+            NowPlayingViewModel.MovieState.RESPONSE_SUCCESS -> showMovies(movieData.data)
+            NowPlayingViewModel.MovieState.RESPONSE_ERROR -> showError()
+            NowPlayingViewModel.MovieState.RESPONSE_LOADING -> showLoading()
+        }
+    }
+
+    private fun showMovies(movieList: List<Movie>) {
+        binding.nowPlayingLoader.visibility = View.GONE
+        binding.imgMain.visibility = View.GONE
+        binding.nowPlayingRecyclerView.layoutManager = GridLayoutManager(context, 3)
```

Comment:
do not hardcode please, you can create a constant in the companion object

Reply:
done